main.py

```
@@ -172,7 +177,10 @@ def save_state():
 
 
 def _safe_chat_id(value):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_safe_chat_id`

(ANN202)